src/applications/personalization/profile/mocks/server.js

```diff
@@ -27,13 +27,15 @@ const responses = {
   'PUT /v0/profile/preferred_names': handlePutPreferredNameRoute,
   'PUT /v0/profile/gender_identities': handlePutGenderIdentitiesRoute,
   'GET /v0/profile/full_name': {
-    id: '',
-    type: 'hashes',
-    attributes: {
-      first: 'Mitchell',
-      middle: 'G',
-      last: 'Jenkins',
-      suffix: null,
+    data: {
+      id: '',
+      type: 'hashes',
+      attributes: {
+        first: 'Mitchell',
+        middle: 'G',
+        last: 'Jenkins',
+        suffix: null,
+      },
```

Comment:
this mock response needed to be nested inside `data` to load correctly. This removes the alert that would display on the profile saying that all info couldn't load fully.